opal/mca/btl/tcp/btl_tcp_component.c

```diff
@@ -729,7 +732,9 @@ static int mca_btl_tcp_component_create_instances(void)
         char* if_name = *argv;
         int if_index = opal_ifnametokindex(if_name);
         if(if_index < 0) {
-            BTL_ERROR(("invalid interface \"%s\"", if_name));
+            opal_show_help("help-mpi-btl-tcp.txt", "invalid if_inexclude",
```

Comment:
Ya https://github.com/mohanasudhan/ompi/blob/master/opal/mca/btl/tcp/help-mpi-btl-tcp.txt#L18 the help message says "An invalid value was given for btl_tcp_if_%s" refering both [in|ex]clude.